web-client/src/ducks/timeline/types.ts

```diff
@@ -2,18 +2,19 @@ import { TimelineItem } from 'src/models/requests/timeline';
 import createActionTypeFactory from 'src/store/utils/createActionTypeFactory';
```

Comment:
Added types needed by other ducks code.    I noticed there is no RESET_POST_WITH_OFFERS_AND_TIMELINE_ITEMS.  This is needed. "_ITEMS" can be dropped.  OBSERVE_TIMELINE can be removed.  GET_TIMELINES_FOR_POST and RESET_TIMELINES_FOR_POST can be removed.